app/views/hyrax/base/_form.html.erb

```diff
@@ -1,31 +1,36 @@
-<%= simple_form_for [main_app, @form],
-                    html: {
-                      data: { behavior: 'work-form',
-                              'param-key' => @form.model_name.param_key },
-                      multipart: true
-                    } do |f| %>
-  <% if f.object.errors.any? %>
-    <div class="alert alert-danger alert-dismissable" role="alert">
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-      <%= f.object.errors.full_messages_for(:base).send(SimpleForm.error_method) %>
-      <%= render 'form_in_works_error', f: f %>
-      <%= render 'form_ordered_members_error', f: f %>
-      <%= render 'form_collections_error', f: f %>
-      <%= render 'form_visibility_error', f: f %>
-    </div>
-  <% end %>
-  <% if Flipflop.batch_upload? && f.object.new_record? %>
-    <% provide :metadata_tab do %>
-      <p class="switch-upload-type">To create a separate work for each of the files, go to <%= link_to "Batch upload", hyrax.new_batch_upload_path %></p>
+<div class="simple_works_form">
```

Comment:
There's no need to add a new `div` because the `form` tag is already acting as a wrapper.